resources/homegrown/data-clinic-for-general.qmd

```diff
@@ -1,76 +1,258 @@
+
+
+### Process once a request is recieved
+
+
+Once a request has been received, it shall be screened by a member of the Data Clinic team. It will then be allocated to follow one of two pathways:  
+
+1. The request can be served by an existing report or we have a data extraction script already prepared that serves the purpose 
+   - Example 1: A Requestor wants a list of all patients discharged from ICU in last month. This can be be served by an existing discharge from ICU report on EPIC.   
+   - Example 2: A Requestor is performing the second cycle of a QIP and would like to re-collect data to assess the impact of their change. If the Data Clinic performed the 1st cycle data extraction, we may be able to reuse the existing data extraction script (with updated dates) to re-collect the data they need post-intervention.   
+   
+   
+In this case, the data request can often be quickly and easily served by the Data analyst without the need for a meeting.   
+   
 
-- Dr Ruaraidh Campbell (ruaraidh.campbell@nhs.net)
-- Dr Sarah Vaughan (s.vaughan5@nhs.net)
-- Dr Hrisheekesh Vaidya (h.vaidya@nhs.net)
+2. The request cannot be served by an existing report/data extraction script   
+   - Example 1: A Requestor is conducting a new QIP and would like data on blood gas results performed throughout a patient's ICU stay. This is not covered by existing reports (e.g. ICNARC report) and as it is a new project, we do not have an existing script to perform the data extraction.    
+   
+   
+In this case, it will be necessary for the Requestor to meet with the Data Clinic team to discuss their project and its data requirements. 
+
+
+
+If a request is allocated to pathway "2", then a meeting is arranged between the Requestor and the Data Clinic team. 
+
+At this meeting, we discuss:  
+- The project aims  
+- The data requirements including inclusion/exclusion criteria for patients  
+- How the data will help answer the project aims  
+- What data is actually available on EHRS and how this may affect the project aims/scope  
+- What format the Requestor requires the data to be in (e.g. do they need an excel spreadsheet with raw data, or do they need the data processed into summary tables/graphs/etc)  
+
+
+Following this meeting, the Data Clinic team will prepare the data extraction (or a limited subset of the full extraction).  
+
+Once this is done, the Data Clinic team meet again with the Requestor to show them the data extraction.   
+-  If the Requestor is happy with this data, the Data Clinic team then proceed to a full data extraction and deliver it to the Requestor.   
+-  If the Requestor are not satisfied, the extraction is revised according to the feedback received. Further meetings are then arranged with after each round of extraction until the Requestor until the data provided is adequate.  
```

Comment:
words are jumbled: think it's meant to say 'further meetings are then arranged with the Requestor after each round of extraction, until the data provided is adequate' or something similar